nautobot/utilities/testing/mixins.py

```diff
@@ -214,6 +220,11 @@ def assertQuerysetEqualAndNotEmpty(self, qs, values, *args, **kwargs):
     # Convenience methods
     #
 
+    def standardize_json(self, jsonText):
+        obj = json.loads(jsonText)
```

Comment:
I know I'm being pedantic, but we don't use `camelCase` in Python and our linters will scream.

```suggestion
    def standardize_json(self, data):
        obj = json.loads(data)
```

Reply:
No problem, thanks for clarifying, fixing! This is my first time with Python, so I'm happy to learn the conventions.